src/main/java/frc/robot/constants/Constants2024.java

```diff
@@ -116,8 +116,8 @@ public class Constants2024 extends GameConstants {
     public static final int INTAKE_MOTOR_1_ID = 6;
     public static final int INTAKE_MOTOR_2_ID = 7;
 
-    public static final double INTAKE_MOTOR_1_SPEED = 1;
-    public static final double INTAKE_MOTOR_2_SPEED = 1;
+    public static final double INTAKE_MOTOR_1_SPEED = 0.8;
```

Comment:
Why?

Reply:
During testing running the intake and feeder were browning out If the battery got low. Slowing down the intake fixed this problem and did not seem to impact the speed we collected a piece. However, now that we current limit the drive train, we could dry increasing the speed back.